docs/Guides/Contributing.md

```diff
@@ -162,9 +162,11 @@ the left sidebar. But wait! We are not quite done yet. There are a few more
 baseline settings that should be set before VSCode is ready.
 
 Press `cmd+shift+p` to bring up the VSCode command input prompt. Type `open
-settings (json)` and then choose the same item from the filtered menu. This will
-open a document that is the settings for the editor. Paste the following JSON
-into this document, overwriting any text already present, and save it:
+settings (json)`. Three options will appear in the dropdown: Workspace, Default,
+and User settings. Select the [VSCode Setting](https://code.visualstudio.com/docs/getstarted/settings)
+that works best for you. This will open a document that is the settings for the
```

Comment:
Shouldn't we recommend a specific choice that is in line with setting up this Fastify dedicated editor? I think we should direct the reader to choose "Default" in order to edit the global editor settings. This way their Fastify specific editor will work for all repos in our org.

Reply:
yes, love it! I'll add that now.